client/components/tinymce/plugins/wpcom-autoresize/plugin.js

```diff
@@ -101,6 +125,13 @@ function wcpomAutoResize( editor ) {
 			if ( tinymce.isWebKit && deltaSize < 0 ) {
 				resize( e );
 			}
+
+			if ( e && e.type === 'keyup' ||
```

Comment:
CC: @beaucollins because we worked on code to make the scrollbar "hug the bottom" for Happychat, and there were do's and don'ts involved in that process. 

It would be nice to get this PR merged soon, though, possibly even as is and then improved later in a PR, because we're getting a fair bit of feedback on this specific issue.